doc/langref.html.in

```diff
@@ -6627,8 +6627,36 @@ test "peer type resolution: *const T and ?*T" {
           <li>{#link|Arrays#} and {#link|Vectors#} with len 0, or with an element type that is a zero bit type.</li>
           <li>An {#link|enum#} with only 1 tag.</li>
           <li>A {#link|struct#} with all fields being zero bit types.</li>
-          <li>A {#link|union#} with only 1 field which is a zero bit type.</li>
+          <li>An {#link|union#} with all fields being zero bit types.</li>
       </ul>
+      {#code_begin|test|test_zero_bit_types#}
+const std = @import("std");
+const testing = std.testing;
+
+test "zero-bit primitive types" {
+    inline for (.{
```

Comment:
Avoid use of `inline for` in a section that is not about `inline for`.

Reply:
I removed the test.